config/crs/kubernetes/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
how is this kustomization file used?
do we package cr examples?

Reply:
it is used [here](https://github.com/nikhil-thomas/tektoncd-operator/blob/3bb89e38cdf10aaafe2f138cac9b2c3607d71943/operatorhub/kubernetes/manifests/fetch-strategy-local/kustomization.yaml#L3)

the cr examples will be filled into the [csv stub/seed](https://github.com/nikhil-thomas/tektoncd-operator/blob/3bb89e38cdf10aaafe2f138cac9b2c3607d71943/operatorhub/kubernetes/manifests/bases/tektoncd-operator.clusterserviceversion.template.yaml#L5)

end result will look like [this](https://github.com/nikhil-thomas/tektoncd-operator/blob/3bb89e38cdf10aaafe2f138cac9b2c3607d71943/operatorhub/kubernetes/release-artifacts/bundle/manifests/tektoncd-operator.clusterserviceversion.yaml#L5)

Reply:
got it. the cr examples doesn't have all newly added fields for customization. we  need to update the cr example?

Reply:
we can, this pr is about adding the tooling. the Bundle is just there as an example.. 
the idea is that we need to regerate it with each commit. `make operator-bundle`.

i will add a ci-test (like TEP generated table test) to ensure that each pr has the updated bundles.

so examples, additional metadata can be regenerated at anypoint.

Reply:
and let us add the cr examples in a different patch.